coldfront/core/allocation/tasks.py

```diff
@@ -41,148 +46,169 @@ def update_statuses():
 
-        expire_notification = allocation_obj.allocationattribute_set.filter(
-            allocation_attribute_type__name='EXPIRE NOTIFICATION').first()
-        if expire_notification and expire_notification.value == 'No':
-            continue
+                            
+        if email_receiver_list:
 
-        resource_name = allocation_obj.get_parent_resource.name
+            send_email_template('Your access to resource(s) have expired',
+                        'email/allocation_expired.txt',
+                        template_context,
+                        EMAIL_SENDER,
+                        email_receiver_list
+                        ) 
 
-        allocation_renew_url = '{}/{}/{}/{}'.format(
-            CENTER_BASE_URL.strip('/'), 'allocation', allocation_obj.pk, 'renew')
+            logger.debug(f'Allocation(s) expired email sent to user {user}.')
 
-        project_url = '{}/{}/{}/'.format(CENTER_BASE_URL.strip('/'),
-                                         'project', allocation_obj.project.pk)
+    if EMAIL_ADMINS_ON_ALLOCATION_EXPIRE:
 
-        template_context = {
-            'center_name': CENTER_NAME,
-            'allocation_type': resource_name,
-            'project_renewal_help_url': CENTER_PROJECT_RENEWAL_HELP_URL,
-            'project_url': project_url,
-            'opt_out_instruction_url': EMAIL_OPT_OUT_INSTRUCTION_URL,
-            'signature': EMAIL_SIGNATURE
-        }
+        if admin_projectdict:
 
-        email_receiver_list = []
-        for allocation_user in allocation_obj.project.projectuser_set.all():
-            if (allocation_user.enable_notifications and
-                allocation_obj.allocationuser_set.filter(
-                    user=allocation_user.user, status__name='Active')
-                    and allocation_user.user.email not in email_receiver_list):
-
-                email_receiver_list.append(allocation_user.user.email)
-
-        send_email_template('Allocation to {} has expired'.format(resource_name),
-                            'email/allocation_expired.txt',
-                            template_context,
-                            EMAIL_SENDER,
-                            email_receiver_list
-                            )
-
-        logger.info('Allocation to {} expired email sent to PI {}.'.format(
-            resource_name, allocation_obj.project.pi.username))
+            admin_template_context = {
+                'project_dict': admin_projectdict,
+                'allocation_dict': admin_allocationdict,
+                'signature': EMAIL_SIGNATURE
+            }  
+            
+            send_email_template('Allocation(s) have expired',
+                                'email/admin_allocation_expired.txt',
+                                admin_template_context,
+                                EMAIL_SENDER,
+                                [EMAIL_TICKET_SYSTEM_ADDRESS,]
```

Comment:
Let's send the admin email to the `EMAIL_ADMIN_LIST` instead of the ticket system address.